packages/docs/docs/guides/6-hosting-a-node/0-intro.md

```diff
@@ -1,24 +1,50 @@
+---# Introduction
 
----
+Now you should be comfortable with features of the Request network.
 
-# Introduction
+In this guide we will explain what is the Request Node and help you to run your own.
 
-Now you should be comfortable with features of the Request network.
+## What is the Request Node?
+
+Request Nodes are servers that run the lower layers of the Request protocol. They connect to the Ethereum and IPFS networks, to store and retrieve request transactions. Protocol users can interact with the Request Node through HTTP using the [request-client.js](../5-request-client/0-intro) library.
+
+## Why run your own Node?
+
+Running the Node locally on your machine will allow you to test your code using the Request Client easilly.
+
+You may also want to host your Node in a server. Hosting your Node is the most decentralized setup possible. It allows you to:
+
+- Store your own data and make sure it is safelly backed up
+- Be technically independent: own your servers and control how you manage them
+- Use custom configuration settings
+- Customize storage options
+
+## How to run your Node?
+
+There are 3 currently supported ways to run the Request Node:
+
+- Run from [**Docker**](./1-docker). Probably the easiest way to run the Request Node.
+- Run the [**code**](./2-code) from the git repository. Especially useful if you are making changes to the protocol layers.
+- Use our kubernetes **helm** charts. The best solution if you want to host your node on a Kubernetes cluster.
+
+On the next pages, you can find out detailed steps on how to run each one of these.
+
+## Prerequisites
 
-In this guide we will guide through the installation of Request node.
+Request uses IPFS and Ethereum as the storage for our protocol data. For this reason the Node needs connections to an Ethereum node and an IPFS node.
 
-## What is it?
+### Ethereum node
 
-Request Nodes are HTTP servers used to allow clients to communicate with the Request Protocol, these servers abstract the complexity of IPFS and Ethereum for the users. The clients can perform all actions on requets by sending messages to the Node via HTTP.
+You can use any HTTP/S Ethereum node to run your Request Node.
+For local development, you can use ganache (explained in more details on the following pages).
 
-## Why hosting your own node?
+An easy way to get going with Ethereum Mainnet or Rinkeby is to use services like infura, that will expose an Ethereum Node API for you.
 
-Hosting your node is the most decentralized setup possible. It allows you to:
-* Be technically independent in your usage of the network
-* Customize your broadcasting latency TODO:check with Vincent the wording for avoiding retention
-* Customize storage options
+### IPFS Node
 
-This option is for advanced developers who are already at ease with Ethereum, IPFS and Request.
+Request uses a dedicated IPFS network. This means that you need to have an IPFS Node with our network settings connected to your Request Node.
```

Comment:
Don't we explain what is a dedicated IPFS network somewhere ? We should leave a link for the user since this is quite a advanced notion.

Maybe use this link ? https://medium.com/pinata/dedicated-ipfs-networks-c692d53f938d